### PR TITLE
doc: add governance document

### PR DESCRIPTION
**Pull request by ThibaultFy**
Signed-off-by: ThibaultFy <50656860+ThibaultFy@users.noreply.github.com>

### CODE REVIEW

File: docs/source/additional/governance.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Process
+-------
+
+We generally use the GitHub issue tracker and pull requests for managing changes.
+For details, including contribution requirements, see our Contributing Guide (ADD LINK).
+Note that we emphasize two-person review for anything other than low-risk contributions.
+
+Issues that we have determined are especially important, particularly if they will take a while, are added to the "next" milestone
+(which identifies "what should be prioritized next").
```
Comment (oleobal) [inline docs/source/additional/governance.rst:31]: This isn't really true, is it?

File: docs/source/additional/governance.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The technical lead has commit rights on the software, and administrative rights to the production site, and can add or remove those rights to others.
+Those with code owners rights can accept changes (typically pull requests) submitted by others.
+These changes include changes to the process and contribution requirements.
+
+Process
+-------
+
+We generally use the GitHub issue tracker and pull requests for managing changes.
+For details, including contribution requirements, see our Contributing Guide (ADD LINK).
```
Comment (oleobal) [inline docs/source/additional/governance.rst:27]: `(ADD LINK)` left over (same lower down)

File: docs/source/additional/governance.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+(which identifies "what should be prioritized next").
+
+We expect people to focus on improving the project, not attacking other
+people. Please strive to "Be excellent to each other."
+For more information, see our Code of Conduct(ADD LINK).
+
+Current people
+--------------
+
+The current Substra Project technical lead is Samuel Lesuffler (@samlesu).
```
Comment (oleobal) [inline docs/source/additional/governance.rst:40]:
```suggestion
The current Substra Project technical lead is Samuel Lesuffleur (@samlesu).
```

**Comment by ThibaultFy** [comment]
Closing it due to no activity

**Pull request closed without merging**